packages/core/package.json

```diff
@@ -36,8 +36,8 @@
     "nsfw": "^1.2.9",
     "p-debounce": "^2.1.0",
     "perfect-scrollbar": "^1.3.0",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
+    "react": "16.8",
```

Comment:
it should be `^16.8.0`

Reply:
Thanks! I have made the change but know the CI is failing and the following commands also seem to be failing:
- `yarn`.
- `yarn --cwd examples/browser start ../.. --hostname=0.0.0.0`.

I'm not able to figure out what I have broken.